### PR TITLE
pylint - partial fix up

### PR DESCRIPTION
**Pull request by pavoljuhas**
- fix wildcard-import and unused-wildcard-import
- fix unused-variable
- fix unnecessary-pass and format
- fix redefined-builtin
- fix anomalous-backslash-in-string
- fix consider-using-f-string
- fix dangerous-default-value

Partially addresses #210

### CODE REVIEW

**Comment by pavoljuhas** [comment]
@dstrain115 - PTAL

File: unitary/quantum_chess/mcpe_utils_test.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def test_effect_of_swap():
 
 
 def test_distance_fn():
-    a1, a2, a3, b1, b2, b3 = cirq.GridQubit.rect(2, 3)
+    a1, a2, a3, _b1, b2, _b3 = cirq.GridQubit.rect(2, 3)
```
Comment (dstrain115) [inline unitary/quantum_chess/mcpe_utils_test.py:60]: what's with underscores on some but not all of them?  Can we make it consistent?0.
same comment in unitary/quantum_chess/swap_updater_test.py

**Review by dstrain115 — APPROVED** [review]

File: unitary/quantum_chess/mcpe_utils_test.py (hunk 1 above)
Comment (pavoljuhas) [reply unitary/quantum_chess/mcpe_utils_test.py:60]: These were flagged by the unused-variable pylint checker.
I can change them to `_` if you think it is more readable.

Comment (dstrain115) [reply unitary/quantum_chess/mcpe_utils_test.py:60]: Yeah, let's change them all to use underscores, just to be consistent.  Thanks for the cleanup!

Comment (pavoljuhas) [reply unitary/quantum_chess/mcpe_utils_test.py]: Sure!  Done in d470f10f23d2e8d8f712cf4dce539c5a8144a3db.

**Pull request merged**